ipatests/prci_definitions/nightly_latest_sssd.yaml

```diff
@@ -0,0 +1,241 @@
+topologies:
+  build: &build
+    name: build
+    cpu: 2
+    memory: 3800
+  master_1repl: &master_1repl
+    name: master_1repl
+    cpu: 4
+    memory: 6450
+  master_1repl_1client: &master_1repl_1client
+    name: master_1repl_1client
+    cpu: 4
+    memory: 7400
+  ipaserver: &ipaserver
+    name: ipaserver
+    cpu: 2
+    memory: 2400
+  ad_master_2client: &ad_master_2client
+    name: ad_master_2client
+    cpu: 4
+    memory: 12000
+  master_3client: &master_3client
+    name: master_3client
+    cpu: 5
+    memory: 10150
+  adroot_adchild_adtree_master_1client: &adroot_adchild_adtree_master_1client
+    name: adroot_adchild_adtree_master_1client
+    cpu: 8
+    memory: 14500
+
+jobs:
+  sssd-fedora/build:
+    requires: []
+    priority: 100
+    job:
+      class: Build
+      args:
+        copr: '@sssd/nightly'
+        git_repo: '{git_repo}'
+        git_refspec: '{git_refspec}'
+        template: &ci-master-latest
+          name: freeipa/ci-master-f36
+          version: 0.0.5
```

Comment:
uhm, our f36 latest template version is: "0.0.7" ref: https://github.com/freeipa/freeipa/blob/master/ipatests/prci_definitions/nightly_latest.yaml#L54

Reply:
Updated.